tests/models/test_google.py

```diff
@@ -1324,71 +1326,8 @@ async def test_google_model_code_execution_tool(allow_model_requests: None, goog
     )
 
     result = await agent.run('What day is tomorrow?', message_history=result.all_messages())
-    assert result.all_messages() == snapshot(
+    assert result.new_messages() == snapshot(
```

Comment:
why drop the all messages?

Reply:
To not have a bunch of duplicated messages from the first run